console/console-init/ui/src/modules/iot-device-detail/components/CredentialsView/CredentialsView.tsx

```diff
@@ -165,9 +159,7 @@ export const CredentialsView: React.FC<ICredentialsViewProps> = ({
                   onConfirmPassword={onConfirmPassword}
                 />
                 {index < credentials.length - 1 && (
-                  <Divider
-                  // className={styles.devider_margin}
-                  />
+                  <Divider className={css(styles.devider_margin)} />
```

Comment:
Please use stylesheets for this.

Reply:
not sure, are you talking about variable name styles ? I'm aready using StyleSheet.create.